net-misc/babeld/babeld-1.9.1.ebuild

```
@@ -1,7 +1,7 @@
-# Copyright 1999-2020 Gentoo Authors
+# Copyright 1999-2023 Gentoo Authors
 # Distributed under the terms of the GNU General Public License v2
 
-EAPI="6"
+EAPI="8"
 
 DESCRIPTION="a loop-free distance-vector routing protocol"
```

Try to remove "A ..." in DESCRIPTION when you can, like:
```suggestion
DESCRIPTION="Loop-free distance-vector routing protocol"
```

---

Will do.